docs/quickstart-aks.md

```diff
@@ -64,6 +70,22 @@ In this section, you will deploy an Azure Kubernetes Service (AKS) cluster using
 
   Provisioning a AKS cluster can take a long time to complete. Please be patient while the request is being processed.
 
+  NOTE: if the creation of the AKS cluster fails with an error like this:
+
+  ```
+  Operation failed with status: 'Bad Request'. Details: Service principal clientID: <SOME_CLIENT_ID>
+  not found in Active Directory tenant <SOME_TENANT_ID>, Please see https://aka.ms/acs-sp-help for more details.
+  ```
+
+  even when retrying the operation multiple times, try this workaround:
+
+  ```bash
+  rm -fr ~/.azure
+  az login
+  ```
+
+  This will re-create your Azure CLI environment. Then, re-start this procedure from the beginning.
+
```

Comment:
How about replacing this section with the following?

>  **Troubleshooting**:
>
>  If you notice the following error message while the cluster is being provisioned, it could indicate the Azure authentication token has expired.
>
>  ```
>  Operation failed with status: 'Bad Request'. Details: Service principal clientID: <SOME_CLIENT_ID> 
> not found in Active Directory tenant <SOME_TENANT_ID>, Please see https://aka.ms/acs-sp-help for more details.
>  ```
>  
>  Please remove the Azure profile directory with `rm -rf ~/.azure` and retry again by following the steps listed in [setting up the cluster](#step-1-set-up-the-cluster).

Reply:
+1. Please also think about whether you want troubleshooting at specific points in the doc, or a separate section at the end with all troubleshooting info merged.